library/src/com/google/maps/android/kml/KmlRenderer.java

```diff
@@ -825,13 +852,16 @@ private void addGroundOverlayInContainerGroups(String groundOverlayUrl,
 
         private final String mIconUrl;
 
+        private final String mModifiedUrl;
```

Comment:
I don't like how this is being stored essentially twice in a slightly different way. "Modified URL" is very generic and ambiguous as to how it's been modified too. Can we just use `mIconUrl` and format it with or without the / where needed?